samples/OpenIdConnectSample/Startup.cs

```diff
@@ -39,10 +37,8 @@ public void ConfigureServices(IServiceCollection services)
                 sharedOptions.SignInScheme = CookieAuthenticationDefaults.AuthenticationScheme);
         }
 
-        public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory)
```

Comment:
I opened a ticket to track that: https://github.com/aspnet/Templates/issues/586.

In the meantime, I'll revert this change.